spec/lib/lighthouse/facilities/client_spec.rb

```diff
@@ -0,0 +1,151 @@
+          'zip' => '01302'
+        }
+      },
+      phone: {
+        'after_hours' => '000-000-0000',
+        'enrollment_coordinator' => '632-550-3888 x3780',
+        'fax' => '632-310-5962',
+        'main' => '632-550-3888',
+        'patient_advocate' => '632-550-3888 x3716',
+        'pharmacy' => '632-550-3888 x5029'
+      },
+      hours: {
+        'friday' => '730AM-430PM',
+        'monday' => '730AM-430PM',
+        'saturday' => 'Closed',
+        'sunday' => 'Closed',
+        'thursday' => '730AM-430PM',
+        'tuesday' => '730AM-430PM',
+        'wednesday' => '730AM-430PM'
+      },
+      services: { 'health' => %w[Audiology Cardiology Dermatology EmergencyCare Gastroenterology
+                                 MentalHealthCare Ophthalmology Orthopedics PrimaryCare SpecialtyCare],
+                  'last_updated' => '2020-04-06', 'other' => [] },
+      feedback: {
+        'effective_date' => '2019-06-20',
+        'health' => { 'specialty_care_routine' => 0.9100000262260437 }
+      },
+      access: {
+        'effective_date' => '2020-04-06',
+        'health' => [
+          { 'established' => 14.352941, 'new' => 158.0, 'service' => 'Audiology' },
+          { 'established' => 34.034482, 'new' => 66.75, 'service' => 'Cardiology' },
+          { 'established' => 3.4, 'new' => 123.5, 'service' => 'Dermatology' },
+          { 'established' => nil, 'new' => 208.0, 'service' => 'Gastroenterology' },
+          { 'established' => 24.228571, 'new' => 134.222222, 'service' => 'MentalHealthCare' },
+          { 'established' => 10.111111, 'new' => 154.6, 'service' => 'Ophthalmology' },
+          { 'established' => 25.17647, 'new' => 122.0, 'service' => 'Orthopedics' },
+          { 'established' => 18.927083, 'new' => 28.8125, 'service' => 'PrimaryCare' },
+          { 'established' => 19.22807, 'new' => 75.317073, 'service' => 'SpecialtyCare' }
+        ]
+      },
+      mobile: false,
+      active_status: 'A',
+      visn: '21',
+      operating_status: { 'code' => 'NORMAL' },
+      facility_type_prefix: 'vha',
+      unique_id: '358'
+    }
+  end
+
+  it 'is an Facilities::Client object' do
+    facilities_client = described_class.new('fake_key')
+    expect(facilities_client).to be_an(Lighthouse::Facilities::Client)
+    expect(facilities_client).to have_attributes(headers: { 'apikey' => 'fake_key' })
+  end
+
+  context 'with an http timeout' do
+    it 'logs an error and raise GatewayTimeout' do
+      allow_any_instance_of(Faraday::Connection).to receive(:get).and_raise(Faraday::TimeoutError)
+      expect do
+        Lighthouse::Facilities::Client.new('fake_key').get_facilities(params)
+      end.to raise_error(Common::Exceptions::GatewayTimeout)
+    end
+  end
+
+  context 'with a bad API key' do
+    it 'returns a 401 error' do
+      VCR.use_cassette('/lighthouse/facilities_401', match_requests_on: %i[path query]) do
+        expect { Lighthouse::Facilities::Client.new('bad_key').get_by_id('vha_358') }
```

Comment:
is there a reason this creates a new facilities client with the api key of bad_key?
since your loading a tape you could use `facilities_client`